tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +334,51 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_multistage_dockerfile(dfobj):
+    """Given a dockerfile object, return the index(es) of FROM line(s)
+    in the dfobj structure."""
+    from_lines = []
+    for idx, st in enumerate(dfobj.structure):
+        if st['instruction'] == 'FROM':
+            from_lines.append(idx)
+    return from_lines
+
+
+def get_multistage_image_dockerfiles(dfobj_multi):
+    """Given a multistage dockerfile object, return a list of structures
+     for building image."""
+    file_path_list = []
+    structure = []
+    file_idx = 0
+    from_lines = check_multistage_dockerfile(dfobj_multi)
+    # Pop the first FROM
+    from_lines.pop(0)
+    # Get the temp folder path
+    temp_folder_path = os.path.dirname(dfobj_multi.filepath) + '/dftemp'
```

Comment:
Can you move the constant `dftemp` to `tern/utils/constants.py`?

Reply:
Do you mean `temp_folder_path = os.path.dirname(dfobj_multi.filepath) + constans.dftemp_dir` 
and in `tern/utils/constants.py` , `dftemp_dir='/dftemp'`

Reply:
In `tern/utils/constants.py` enter:
```
# temporary directory for multistage Dockerfile analysis
multistage_dir = 'dftemp'
```
And then in this line
`temp_folder_path` = os.path.join(os.path.dirname(dfobj_multi.filepath), multistage_dir)`

Reply:
Got it!